resources/js/components/settings/Genders.vue

```diff
@@ -270,7 +270,7 @@ export default {
 
       createForm: {
         name: '',
-        type: '',
+        type: 'M',
```

Comment:
There is a notion of "default gender" in Monica, and it should be used here instead of hard-code 'M' as the default.

Reply:
Also see #4753.